llvm/lib/Target/AMDGPU/AMDGPUTargetMachine.cpp

```diff
@@ -596,6 +596,7 @@ static bool mustPreserveGV(const GlobalValue &GV) {
   if (const Function *F = dyn_cast<Function>(&GV))
     return F->isDeclaration() || F->getName().startswith("__asan_") ||
            F->getName().startswith("__sanitizer_") ||
+           F->getName() == "__divti3" ||
```

Comment:
we're stuck preserving this in the IR at all times which isn't really ideal

Reply:
will add a function attribute for that